portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java

```
@@ -0,0 +1,37 @@
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for listing comments. */
+@WebServlet("/delete-data")
+public class DeleteCommentsServlet extends HttpServlet {
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Query query = new Query("Comment");
+        DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
```

Not something to implement right now, but something to think for future implementations(capstone maybe?).  Any operations involving external systems could be error prone and we should handle it gracefully.  For example, in this case think the following.

1) How can you handle this gracefully when the delete fails?
2) Can we return an error that the user can understand over some exception which exposes our service internals sometimes.
3) Can we retry the delete?

This is very important while building fault tolerant systems.